packages/react/src/components/pickers/BasePicker.tsx

```diff
@@ -350,17 +369,53 @@ export class BasePicker<T extends {}, P extends IBasePickerProps<T>>
             )}
           </div>
         </SelectionZone>
+        {this.renderError(classNames.error)}
         {this.renderSuggestions()}
       </div>
     );
   }
 
+  protected _getDescribedBy = (items: T[], hasError: boolean): string => {
+    const describedBy = [];
+    if (items.length > 0) {
+      describedBy.push(this._ariaMap.selectedItems);
+    }
+    if (hasError) {
+      describedBy.push(this._ariaMap.error);
+    }
+    return describedBy.join(' ');
+  };
+
   protected canAddItems(): boolean {
     const { items } = this.state;
     const { itemLimit } = this.props;
     return itemLimit === undefined || items.length < itemLimit;
   }
 
+  protected renderLabel(inputId: string, className?: string): JSX.Element | null {
+    const { label, disabled, required } = this.props;
+    if (!label) {
+      return null;
+    }
+    return (
+      <Label className={className} disabled={disabled} required={required} htmlFor={inputId}>
+        {label}
+      </Label>
+    );
+  }
+
+  protected renderError(className?: string): JSX.Element | null {
+    const errorMessage = this.props.errorMessage || this.state.errorMessage;
```

Comment:
Should these be the other way around?

```suggestion
    const errorMessage = this.state.errorMessage ?? this.props.errorMessage;
```

Reply:
I copied the implementation from the `_errorMessage` property of the TextField.base.tsx